app/lib/activitypub/tag_manager.rb

```diff
@@ -85,7 +85,7 @@ def local_uri?(uri)
     !host.nil? && (::TagManager.instance.local_domain?(host) || ::TagManager.instance.web_domain?(host))
   end
 
-  def uri_to_local_id(uri, param = :id)
+  def uri_to_local_id(uri, param)
```

Comment:
Why did you change this?

Reply:
Probably I thought `:id` is Status specific, and it shouldn't be defaulted. but now it looks generic term. I've removed this change.